src/main/CommandLine.cpp

```diff
@@ -763,23 +817,26 @@ handleCommandLine(int argc, char* const* argv)
           runWriteQuorum},
          {"version", "print version information", runVersion}}};
 
-    auto command = commandLine.selectCommand(argc, argv);
+    auto ajustedCommandLine = commandLine.adjustCommandLine({argc, argv});
+    auto command = commandLine.selectCommand(ajustedCommandLine.first);
     if (!command)
     {
         return nullopt<int>();
     }
 
+    auto exeName = argv[0];
+    auto commandName = fmt::format("{0} {1}", argv[0], command->name());
```

Comment:
I would not bother with `argv[0]` in case it has spaces

Reply:
I don't think we ever extract the binary path, so we're probably good to just ditch this entirely

Reply:
what do you mean exactly? to make commandName = command->name() ?

Reply:
I'd say put a hard-coded string such as "stellar-core" instead of `argv[0]`

Reply:
updated